src/test/rustdoc-ui/feature-gate-output-format.stderr

```diff
@@ -0,0 +1,2 @@
+error: the -Z unstable-options flag must be passed to enable --output-format for documentation generation
```

Comment:
This appears to be an expected output without the associated test, I'm guessing the one you removed due to issues. should the expected output also get removed, as I don't think this actually does anything?

Reply:
Oops, good catch. I removed it and added a run-make test :)